templates/javascript-ethers-hardhat/scripts/whitelist-reader.js

```diff
@@ -5,20 +5,20 @@ const { getServiceData, whitelistBeaconReader } = require("@api3/services");
 async function main() {
   const network = hre.network.name;
 
-  // TODO: Update the mocked contract so that whitelisting is necessary
-  if (
-    network.toLowerCase() === "hardhat" ||
-    network.toLowerCase() === "localhost"
-  )
+  if (network.toLowerCase() === "localhost") {
```

Comment:
By looking at this I'm assuming the idea is to disable hardhat network, am I correct?

Reply:
Yes, it just complicates things... Apart from tests, I'd like rather assume that thing doesn't exist.